flixel/addons/effects/IFlxEffect.hx

```diff
@@ -0,0 +1,12 @@
+package flixel.addons.effects;
+import openfl.display.BitmapData;
+import openfl.geom.Point;
+
+interface IFlxEffect
+{
+	public var active:Bool;
+	public var offset:Point;
+	public function destroy():Void;
```

Comment:
This line should be removed, instead this interface should extend `IFlxDestroyable`.